Bottom: 0e0998849ef93053e68fcec1460996a6921d4f76
Top:    d689c76ff2b459466819993b1719e6b8d3abe886
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-01-12 22:30:08 +0000

Refresh of set-changed-flag-on-pcb-when-l

---

diff --git a/src/action.c b/src/action.c
index affefba..700bf29 100644
--- a/src/action.c
+++ b/src/action.c
@@ -1150,6 +1150,7 @@ NotifyMode (void)
 	     */
 	    DrawElement (element);
 	    Draw ();
+	    SetChangedFlag (true);
 	    hid_actionl ("Report", "Object", NULL);
 	  }
 	else if (type != NO_TYPE)
@@ -1164,6 +1165,7 @@ NotifyMode (void)
 		DrawObject (type, ptr1, ptr2);
 		Draw ();
 	      }
+	    SetChangedFlag (true);
 	    hid_actionl ("Report", "Object", NULL);
 	  }
 	break;
